lib/src/executables/react16_post_rollout_cleanup.dart

```diff
@@ -48,9 +48,11 @@ void main(List<String> args) {
   );
 
   if (hasUnaddressedReact16Comment(query, logger: logger)) {
-    throw Exception('There are still unaddressed comments from the '
-        'React 16 upgrade codemod. These should be addressed before cleaup is'
-        ' attempted.');
+    logger.severe(
+        'There are still unaddressed comments from the React 16 upgrade codemod. '
+        'These should be addressed before cleanup is attempted.');
+    exitCode = 64;
```

Comment:
Why 64?

Reply:
It's the `usage` error code. My thought was that they're using the codemod incorrectly if they still have outstanding comments to address, so out of the specific error codes it fits the best 🤷‍♂

Reply:
I'm not sure I agree there. From https://www.freebsd.org/cgi/man.cgi?query=sysexits&apropos=0&sektion=0&manpath=FreeBSD+4.3-RELEASE&format=html
> EX_USAGE (64)
> The command was used incorrectly, e.g., with the
> wrong number of arguments, a bad flag, a bad syntax
> in a parameter, or whatever.

This case will happen during normal running of the command (e.g., running codemod for the first time, running again before going in and addressing all comments), and doesn't seem like incorrect usage.

Can we switch to `1` to match the others?

Reply:
Yep, definitely! 😄